invoice/invoice.py

```diff
@@ -86,12 +85,11 @@ def populate_columns(invoice_df: pd.DataFrame, server_types_data: List) -> pd.Da
 
     for key in ["cores", "memory", "disk", "prices"]:
         invoice_df[key] = [
-            str(get_server_information(x, server_types_data, key)) for x in server_type
+            str(get_server_information(x, server_types_data, key)) if x else None
```

Comment:
could you explain why the for loop is not needed here?

Reply:
the for loop is needed and used. It is just confusing:

str(get_server_information(x, server_types_data, key))
  if x else None
    for x in server_type

for each server_type, if it is not None, it will get the get_server_information. And if is none it will use the value None.

Reply:
my bad, I didn't read the next line correctly